ocaml/idl/datamodel.ml

```diff
@@ -191,6 +222,10 @@ module Session = struct
         ; local_logout
         ; get_all_subject_identifiers
         ; logout_subject_identifier
+        ; get_all_records
+        ; count_sessions
+        ; group_sessions_by_originator
```

Comment:
This is in the Sessions module, so `group_by_originator` is probably good enogh; similar for the other field.